portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,36 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Jesús Urquídez Calvo</h1>
+        <p>I'm a mexican of 19 years old that love Computer Science and Videogames...and a little bit weirdo</p>
+        <p>This is my portfolio.</p>
+
+    <table>
+        <th>
+            <img width="300" style="border: solid;" src="/images/myself.jpg">
+        </th>
+        <th>
+            <p class="text" style="text-align: justify;">I'm from Mexicali, Baja California (is the state below California) and I consider myself someone who never give up. I grew up in a family with low income and always me brother and I had to achieve all of our schoolarships by ourself studying everyday to win every competition and learn a lot of thing.</p>
+            <p class="text" style="text-align: justify;"> And...I got until this point only with the help of all my friends and family and I think that the think that makes me 'me' is that I'm a very obtinate person that never give up</p>
+            <p class="text"> Oh...and hi! to anyone who see this page :3</p>
+        </th>
+    </table>
+
+    <div>
+        <img width="100" src="/images/me_1.jpg">
+        <img width="100" src="/images/me_2.jpg">
+        <img width="100" height="100" src="/images/me_3.jpeg">    
+    </div>
+
+    <div>
+        <a href="nkedin.com/in/jesus-urquidez-calvo-517668194/"><img src="https://img.icons8.com/plasticine/100/000000/linkedin.png"/></a>
```

Comment:
```suggestion
        <a href="linkedin.com/in/jesus-urquidez-calvo-517668194/"><img src="https://img.icons8.com/plasticine/100/000000/linkedin.png"/></a>
```

Is this URL incomplete?

Reply:
Yes, I didn't see the error

Reply:
Even so, it causes an error, I just noticed that it is due to the lack of the HTTP protocol